indico/modules/categories/templates/display/overview/base.html

```diff
@@ -43,6 +43,7 @@ <h3>{% trans %}Display options{% endtrans %}</h3>
                         </select>
                     </div>
                 </div>
+                {% block cat_filter_extra %}{% endblock %}
```

Comment:
Maybe the name could be improved a bit? `category_extra_filters`?

Reply:
Sure, if you prefer that, done. I was unable to project a clear block naming convention from the block names I see?

Reply:
No need for a separate commit though ;) You can squash them into one :)

Reply:
Can be squashed on merge,.